server/controllers/solicitations/Create.js

```diff
@@ -26,7 +28,14 @@ module.exports = async function createSolicitation(ctx) {
     }
   }
 
-  const { name, email, registrationNumber, type } = ctx.request.body
+  const {
+    name,
+    email,
+    registrationNumber,
+    type,
+    course,
+    admissionType
+  } = ctx.request.body
 
   const validRequest = [name, email, type].every(item => item !== undefined)
```

Comment:
@lubien veja aqui, é garantido que não dará erro se `registrationNumber, course, ou admissionType` forem undefined?

Vi que são opcionais, não lembro se dá bronca se eu disser que eles devem ser salvos, como ocorre logo abaixo.

Se isso for um problema, penso que deveríamos ter algo como:

```javascrip
const solicitation = ctx.request.body

const validRequest = ['name', 'email', 'type'].every(item => user[item] !== undefined)
```
E no momento de salvar, algo como isso:
```javascrip
ctx.body = await Solicitation.forge(solicitation).save()
```

Reply:
Matrícula é o único opcional